test/indexer/test_tdr.py

```diff
@@ -125,6 +125,11 @@ def _bigquery(self, project: str) -> bigquery.Client:
                                credentials=AnonymousCredentials(),
                                client_options=options)
 
+    def _job_has_result(self, job_info: JSON) -> bool:
+        # Some queries (such as the one in Plugin._downstream_from_files()) can
+        # produce job stats that are empty in the test environment.
```

Comment:
```suggestion
        # Some queries (such as the one in Plugin._downstream_from_files()) can
        # produce job stats that are empty in the test environment.
```

I'm surprised that only some queries lack the job stats. Have you observed BQ emulator returning job stats for any queries? If so, which ones?

Reply:
Yes, all queries return a job stats with two keys, `totalBytesBilled` and `totalBytesProcessed`, except for the `UPDATE` queries for `TestTDRHCAPlugin.test_fetch_bundle`, and a query run for `TestAnvilIndexer.test_list_and_fetch_bundles`.

Interestingly, the latter test runs two similar shaped queries, one which does produce job stats (`_downstream_from_biosamples()`) and one that does not (`_downstream_from_files()`).